Common/AnatupleProducer.py

```diff
@@ -0,0 +1,121 @@
+import ROOT
+import numpy as np 
+import Common.BaselineSelection as Baseline
+import Common.Utilities as Utilities
+import Common.ReportTools as ReportTools
+
+
+
+def createAnatuple(inFile, outFile, period, sample, X_mass, snapshotOptions, isData=0, range=100):
+    Baseline.Initialize(True, True)
+    df = ROOT.RDataFrame("Events", inFile) 
+    df = df.Range(range)
```

Comment:
- pass range as a command line argument
- set default value to None (we don't want to apply range by default)
```python
if range is not None:
  df = df.Range(range)
```